charter.md

```diff
@@ -17,7 +18,17 @@ The SPICE WG will support digital credential formats leveraging existing IETF st
 
 Digital credentials are identity documents with attributes (public or private claims in IANA registries) about the identity. The identifiers referenced could be about any subject, but common examples include people, devices, assets, organizations and processes.
 
-Examples of producing and consuming digital credentials with the "Three Role Model" include credential endorsement by third parties, credential trustworthiness via remote attestation evidence appraisal for issuers, or credential transparency via notarization.
+## Background
+
+The IETF has standardized the Web Authorization (OAuth) protocol to allow a user to grant a third-party web site or application access to the user's protected resources, without necessarily revealing their long-term credentials, or even their identity. The JSON Web Token (JWT) was an important building block that standardized the token format. OAuth also offered fine-grained access to resources based on the resource owner's permissions, which are encoded inside the access token. OAuth and JWTs were re-used by the OpenID Foundation as building blocks for standardizing identity management protocols.
+
+Since OAuth was heavily focused on the HTTP-based web, it required adaptations for use in constrainted Internet of Things environments by using an efficient binary encoding based on CBOR, CBOR Object Signing and Encryption (COSE), and protocols commonly used in those environments, such as MQTT and CoAP. This standardization work was completed in the IETF ACE working group. With the CBOR Web Token (CWT) an alternative token format was developed, which like the JWT, also relied on an IANA-based registry for claims. Advanced privacy features were not considered a high priority in those IoT scenarios, which focused on machine-to-machine communication rather than interactions with humans.
+
+During the last few years there has been renewed interest in privacy-enhancing technologies for identity management systems that utilize technologies offering selective disclosure, data minimization and unlinkability. Foundational building blocks have been developed with BBS Signatures, RSA Blind Signatures, Verifiable Random Functions, and Selective-Disclosure techniques. These technologies have subsequently been incorporated into JOSE and OAuth-based protocols. The integration into protocols using CWTs is, however, missing. This working group aims to close this gap by enabling the secure and privacy-friendly use of CWT-based credentials for use cases that go beyond constrained Internet of Things devices. Envisioned use cases where these credentials will be used include education, digital wallets, business-to-business supply chain interactions, and digital media.
+
```

Comment:
Proposal on the 2024-01-04 call was to move this text to a wiki 
```suggestion

```

Reply:
The proposal on the call was to keep the 3rd paragraph, which I have done in https://github.com/transmute-industries/ietf-spice-charter/pull/17/commits/20e0a541e2acbe5d49100d8f823d74e54bc1bd63

Although it may possibly be refined further